projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts

```diff
@@ -2202,6 +2203,61 @@ describe('IgxGrid Component Tests', () => {
                 expect(editedCell.column.field).toEqual('Downloads');
                 expect(editedCell.inEditMode).toEqual(true);
             }));
+
+            it(`Should update row changes when focus overlay buttons on tabbing`, (async() => {
+                let currentEditCell: IgxGridCellComponent;
+                const targetCell = fixture.componentInstance.focusGridCell(0, 'Downloads');
+                fixture.detectChanges();
+                grid.parentVirtDir.getHorizontalScroll().scrollLeft = 0;
+                await wait(300);
```

Comment:
There is no need wait to be 300, you can just use await wait().
Also there is no need to set scrollLeft, because the column Downloads is the first column.

Reply:
done, thank you